generator/mavgen_python.py

```diff
@@ -785,7 +785,12 @@ def generate_methods(outf, msgs):
     def field_descriptions(fields):
         ret = ""
         for f in fields:
-            ret += "                %-18s        : %s (%s)\n" % (f.name, f.description.strip(), f.type)
+            field_info = ""
+            if f.units: field_info+="%s " % f.units
```

Comment:
Please put the conditional code on a new line.  And spaces around the `+=`.  Generally, make the flake8 output no worse ('though I expect it to be pretty terrible ATM).

Reply:
Done. Thanks. Might push a couple of flake8 fixes through too, but this now ready for review